### PR TITLE
PR: Algo Quick Sort in JavaScript

### PR DESCRIPTION
**Pull request by Mohitraut07**
Issue: #59 issue solved and quick sort algo
This README provides an overview of Quick Sort, explains its operation in simple terms, and includes the JavaScript code for implementing the algorithm.

### CODE REVIEW

**Comment by Abishethvarman** [comment]
@Mohitraut07 chnage the PR title in the correct format. Refer previous merged PR

**Comment by Abishethvarman** [comment]
> Issue: #59 issue solved and quick sort algo

Add proper description by mentioning what algorithm and what language

**Comment by Mohitraut07** [comment]
@Abishethvarman please check and review the PR.

File: 2-algorithms/2-javascript/2-Quick-sort.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Certainly! Here's a README file for the Quick Sort algorithm in JavaScript:
```
Comment (Abishethvarman) [inline 2-algorithms/2-javascript/2-Quick-sort.md:1]: What about this line

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

File: 2-algorithms/2-javascript/2-Quick-sort.md (hunk 1 above)
Comment (Abishethvarman) [reply 2-algorithms/2-javascript/2-Quick-sort.md:1]: remove the line and resolve the issues

[the remark this answers is not in the document]
Comment (Mohitraut07) [reply 2-algorithms/2-javascript/2-Quick-sort.md]: okay

Comment (Mohitraut07) [reply 2-algorithms/2-javascript/2-Quick-sort.md]: will remove it ASAP.

**Review by Abishethvarman — COMMENTED** [review]
ok

**Review by Abishethvarman — APPROVED** [review]
Approved to merge

**Review by Thevapriyan07 — APPROVED** [review]
Approve to merge

**Pull request merged**